### PR TITLE
Improve start instructions in readme, add a debug section

### PR DESCRIPTION
**Pull request by ctmbl**
everything's in the title :slightly_smiling_face:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ API_URL = "https://iscsc.fr/api/articles"
...
 ## Start the bot
-To run it on the host system simply `python3 main.py`.
+### Directly on host system
+To run it on the host system simply run `python3 main.py`.
 
-**But** you really should containerize it thanks to the provided `Dockerfile`, you just have to run `./run.sh` the script clean previous running containers, build the current one and start it.
+:warning: but you **should really** consider using the [containerized deployment](#Containerized-with-Docker).
+
+### Containerized with Docker
+The bot can run in Docker thanks to the provided `Dockerfile`. To ease the starting, you just have to run `./run.sh`: the script cleans previous running containers, builds the current one and starts it.
```
Comment (gbrivady) [inline README.md]: Phrasing is weird here "To ease the starting", maybe just say "To start the bot in docker, [...]"

File: README.md [hunk 2]
```diff
@@ -18,6 +18,17 @@ API_URL = "https://iscsc.fr/api/articles"
...
+
+Once the command is finished, simply run `docker ps` to check that the bot is indeed running, if you can't see a `iscsc.fr-notify-bot:latest` container running check the [debug section](#Debug).
+
+#### Debug
+If the container isn't running after `./run.sh` it has probabbly crashed due to an error, first check that you well set up the required variables, described in the [config section](#Config).
```
Comment (gbrivady) [inline README.md]:
```suggestion
If the container isn't running after `./run.sh` it has probably crashed due to an error, first check that you well set up the required variables, described in the [config section](#Config).  
```
Typo

File: README.md (hunk 1 above)
Comment (ctmbl) [reply README.md]: you're right don't know what I was thinking ahah

Comment (ctmbl) [reply README.md]: fixed [bf05de2](https://github.com/iScsc/iscsc.fr-blog-notify/pull/4/commits/bf05de2dd780769f249ef691f72c233f63ba0bf9)

File: README.md (hunk 2 above)
Comment (ctmbl) [reply README.md]: I commited it :wink:

**Comment by gbrivady** [comment]
Good to go

**Review by gbrivady — APPROVED** [review]
Good to go

**Pull request merged**